tests/unit/store/settings.spec.js

```diff
@@ -42,15 +42,15 @@ describe("store/settings", () => {
     expect(store.state.settings.user_details.uuid).toBeNull();
     expect(store.state.settings.user_details.nickname).toBeNull();
     store.commit("settings/set_user_details", array_of_userid);
-    expect(store.state.settings.user_details[0].user_id).toEqual(0);
-    expect(store.state.settings.user_details[0].uuid).toEqual(
+    expect(store.state.settings.user_details[0].user_id).toStrictEqual(0);
+    expect(store.state.settings.user_details[0].uuid).toStrictEqual(
       "2VSckkBYr2An3dqHEyfRRE"
     );
-    expect(store.state.settings.user_details[0].nickname).toEqual(
+    expect(store.state.settings.user_details[0].nickname).toStrictEqual(
       "User account -1"
     );
   });
-  test("Store a single customer detail can be assigned to the customer_details", () => {
+  test("When initialized customer_details UUID, API-key, customer name and user_details UUID, nickname is empty, Then commit customer_details with valid cust_id, UUID, API-key, customer name and user_details of user_id, UUID, nickname to the Vuex and assert the same", () => {
```

Comment:
similar comment as previous test

Reply:
Would like to retain the present title, its more informative.